App/src/pages/dossier/Dossiers.tsx

```diff
@@ -257,6 +257,7 @@ export default function Dossiers() {
                             handleNavigateToDossierReview={handleNavigateToDossierReview}
                             useIcons={false}
                             reviewIcons={true}
+                            editable={true}
```

Comment:
This is not where we want it. You should not be able to edit your dossiers once they've been submitted. Instead, the page that displays Dossiers Requiring Your Review should be the one with this edit button.

Reply:
Ok, sorry for the error, I fixed it now.

Reply:
I also got rid of the unneeded button with my last commit